lib/hashie/extensions/dash/coercion.rb

```diff
@@ -2,12 +2,14 @@ module Hashie
   module Extensions
     module Dash
       module Coercion
-        # Extends a Dash with the ability to define coercion for properties.
-
-        def self.included(base)
-          base.send :include, Hashie::Extensions::Coercion
+        includer = ::Hashie::Extensions::Coercion::CoercionSystemIncludeBuilder
```

Comment:
I'm down with backward compatibility in general, but I am not sure what the implications are here.